pkg/protocol/recordlayer/fuzz_test.go

```diff
@@ -5,12 +5,14 @@ package recordlayer
 
 import (
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
```

Comment:
Possible to do this with testify/not take on another dependency?

Not a big deal, but users do appreciate keeping deps small :)